examples/ansible/testvcertplaybook.yaml

```diff
@@ -0,0 +1,20 @@
+config:
+  connection:
+    platform: vaas
+    credentials:
+      tokenURL: <Your Service Account Token URL - Copy from VCP> 
+      externalJWT: '{{ Env "TLSPC_EXTERNAL_JWT" }}'
+certificateTasks:
+  - name: myCertificate
+    renewBefore: 31d
+    request:
+      csr: service

```

Comment:
@vertigo-one : Would you mind changing to "local" instead of "service"? I'm trying to encourage a best practice of generating the private keys on the machines, rather than in the platform. (There's no need, it's more secure, and for TLSPC, doesn't require that we have a vSat!)

Reply:
Sure. Not a problem.

Reply:
Updated in bf95fa38a9b528b083446a14790d1543abd39f06